lib/compilers/java.ts

```diff
@@ -77,20 +84,28 @@ export class JavaCompiler extends BaseCompiler {
                         maxOutput: maxSize,
                         customCwd: dirPath,
                     });
-                    const oneResult = {
-                        asm: objResult.stdout,
+                    const oneResult: ParsedAsmResult = {
+                        asm: [
+                            {
+                                text: objResult.stdout,
+                            },
+                        ],
```

Comment:
Unsure if this is correct, just made a best guess for how to put this in a ParsedAsmResult

Reply:
Very good question .. this will need to be tested. But...I wonder how it worked before if this isn't quite right ...

Reply:
I'm cool with finding out by merging and dealing with the consequences

Reply:
With the regression in #4668 I'm wondering how this worked before :thinking: